### DISCUSSION TITLE
Which Model used by current Frigate

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#3268] [General]
Which model does the 11 Beta use? I am assuming one the models from [https://coral.ai/models/object-detection/](https://coral.ai/models/object-detection/)

**Comment by NickM-27** [comment]
The same model as previous versions, from the coral set. It's here https://github.com/blakeblackshear/frigate/blob/release-0.11.0/docker/Dockerfile#L72

**Reply by bm55** [reply]
Thanks!
(ssdlite_mobiledet_coco_qat_postprocess_edgetpu.tflite)
